Ejer3LinqPractice_SecondTry/Program.cs

```diff
@@ -8,8 +10,238 @@ namespace Ejer3LinqPractice_SecondTry
 {
     class Program
     {
+        static List<Car> listaCoche = new List<Car>();
+
+        static List<Car> Ejer1()
+        {
+            using (StreamReader streamReader = new StreamReader("Cars.json"))
+            {
+                string json = streamReader.ReadToEnd();
+                listaCoche = JsonConvert.DeserializeObject<List<Car>>(json);
+                return listaCoche;
+            }
+        }
+
+        static void Ejer2()
+        {
+            var fabricante = listaCoche.GroupBy(x => x.Maker);
+
+            foreach (var item in fabricante)
+            {
+                Console.WriteLine($"Fabricante: {item.Key}");
+            }
+        }
+
+        static void Ejer3()
+        {
+            var colores = listaCoche.GroupBy(x => x.Color).Select(x => x.First());
+
+            foreach (var item in colores)
+            {
+                if (!String.IsNullOrEmpty(item.Color)) // Hay varios colores que estan en null
+                {
+                    Console.WriteLine($"Color: {item.Color} Fabricante: {item.Maker} Modelo: {item.Model}");
+                }
+            }
+        }
+
+        static void Ejer4()
+        {
+            var colorVerde = listaCoche.Where(x => x.Color.Equals("Green"));
+
+            foreach (var item in colorVerde)
+            {
+                Console.WriteLine($"Color: {item.Color} Fabricante: {item.Maker} Modelo: {item.Model}");
+            }
+        }
+
+        static void Ejer5()
+        {
+            // Esto sirve para cambiar el formato decimal punto por coma...
+            var currentCulture = System.Globalization.CultureInfo.InstalledUICulture;
+            var numberFormat = (System.Globalization.NumberFormatInfo)currentCulture.NumberFormat.Clone();
```

Comment:
@cvasilivg Not bad. Creo que no hay que hacer el clone, si el usuario hay usa una coma, que pasaría?